src/main/java/com/beerjournal/breweriana/item/rating/persistence/Rating.java

```diff
@@ -0,0 +1,28 @@
+package com.beerjournal.breweriana.item.rating.persistence;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@Document
+@Data
+@EqualsAndHashCode(exclude = "id")
+@RequiredArgsConstructor(access = PRIVATE)
+public final class Rating {
+    @Id
+    private final ObjectId id;
```

Comment:
In what way do you need to adjust business to tools while using embedded collection? 
Now for storing only one int value you need to have 3 ids. If you use embedded collection 1 id is enough (value, userId).
But ok, it's only university project and we don't have time for discussing. So don't change the model